<launch>
    <!-- Launch the mission node -->
    <include file="$(find sky_sim)/launch/apm.launch" />
    <include file = "$(find sky_sim)/launch/line_follower.launch" />
    <remap from = "/webcam_down/image_raw_down" to="/sky_vision/down_cam/img_raw"/>
    <remap from = "/webcam_down/camera_down_info" to="/sky_vision/down_cam/type"/>
    <node pkg="sky_vision" name="line_detector" type="line2ros.py" output="screen"/>
    <!-- <node name="sky_mission" pkg="sky_mission" type="mission.py" output="screen"/> -->
</launch>